core_lib/web_helpers/web_helprs_utils.py

```diff
@@ -14,7 +14,7 @@ def init(server_type: ServerType):
         WebHelpersUtils.server_type = server_type
 
     @staticmethod
-    def get_server_type():
+    def get_server_type() -> server_type:

```

Comment:
did you mean to return `WebHelpersUtils.ServerType` instead?

what kind of type is it `server_type`?

Reply:
it is returning the `server_type`  which will have the actual value, but `ServerType` is the enum class that's why i went with `server_type`